org.upscayl.app.yml

```diff
@@ -0,0 +1,70 @@
+id: org.upscayl.app
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: upscayl-run
+separate-locales: false
+
+finish-args:
+  - --allow=devel
+  - --allow=multiarch
+  - --device=dri
+  - --filesystem=home
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+
+modules:
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        tag: v2022.04
+  - name: upscayl
+    buildsystem: simple
+    cleanup:
+      - upscayl-2.0.1-linux
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: >-
+          https://github.com/upscayl/upscayl/releases/download/v2.0.1/upscayl-2.0.1-linux.zip
+        sha256: 96883ae1bb9a3a5c0972ab7e402c5132e40c4fe0ffaeec8753524df16b03de51
+      - type: file
+        path: org.upscayl.app.metainfo.xml
+      - type: file
+        path: icon.png
+      - type: file
+        path: org.upscayl.app.desktop
```

Comment:
Also I don't see the desktop file in your submission.

Reply:
Ah sorry, forgot to include it